### PR TITLE
Cleanup after environments renaming

### PR DESCRIPTION
**Pull request by dariadomagala**
**Description**

Changes proposed in this pull request:

- Cleanup after environments renaming

**Related issue(s)**

### CODE REVIEW

File: core/src/luigi-config/luigi-config.js [hunk 1]
```diff
@@ -704,9 +704,9 @@ Promise.all([getBackendModules(), getSelfSubjectRulesReview()])
                     hideFromNav: true,
                     children: [
                       {
-                        pathSegment: ':environmentId',
+                        pathSegment: ':namespaceId',
                         context: {
-                          environmentId: ':environmentId'
+                          namespaceId: ':namespaceId'
```
Comment (JohannesDoberer) [inline core/src/luigi-config/luigi-config.js]: For backward compatibility I suggest to send the environmentId in the context obj as well.
`context:{
   environmentId: ':environmentId'
   namespaceId: ':namespaceId'
}`
You never know if other mf's still use environmentId

**Comment by JohannesDoberer** [comment]
LGTM :)

**Comment by dariadomagala** [comment]
@pkosiec https://github.com/kyma-project/kyma/pull/3234 :) tests are green

**Pull request merged**